libmscore/measure.cpp

```diff
@@ -2792,15 +2792,14 @@ Fraction Measure::stretchedLen(Staff* staff) const
 //   cloneMeasure
 //---------------------------------------------------------
 
-Measure* Measure::cloneMeasure(Score* sc, TieMap* tieMap)
+Measure* Measure::cloneMeasure(Score* sc, int tick, TieMap* tieMap)
       {
       Measure* m      = new Measure(sc);
       m->_timesig     = _timesig;
       m->_len         = _len;
       m->_repeatCount = _repeatCount;
 
-      for (MStaff* ms : _mstaves)
-            m->_mstaves.push_back(new MStaff(*ms));
+      Q_ASSERT(sc->staves().size() >= _mstaves.size()); // destination score must we're cloning into have at least as many staves as measure being cloned
```

Comment:
This comparison causes a compiler warning because the size() functions for QList and std::vector return different types. Also, the word "must" in the comment is in the wrong place. I would change it to
```
      Q_ASSERT(sc->staves().size() >= int(_mstaves.size())); // destination score we're cloning into must have at least as many staves as measure being cloned
```

Other than that, I am fine with this.

As for the original for loop, it was incorrect because it added MStaff objects based on the old measure's score to the new measure's _mstaves list. And the Measure constructor already adds MStaff objects based on the correct score to the measure's _mstaves list.

Reply:
@mattmcclinch fixed this. Thank you for helping me look into this yesterday.